python/py-ipaddress/Portfile

```diff
@@ -11,7 +11,7 @@ platforms           {darwin any}
 license             PSF
 supported_archs     noarch
 
-python.versions     27 35 36 37 38 39
+python.versions     27 35 36 37 38 39 310 311
```

Comment:
it appears to met that there are no dependents for the py35/py36/py37 subports ; can you please remove those?

Reply:
ping @barracuda156 - if you address this comment I'll happily merge the PR.

Reply:
@reneeotten Yes, I will today. (Sorry for a delay, just arrived yesterday night, and there is much stuff to sort.)